src/Components/Details/Details.jsx

```diff
@@ -9,8 +9,8 @@ import { getServer } from '../../helpers';
 
 const Details = () => {
   const [productStyles, setProductStyles] = useState([]);
-  const [currentProductStyle, setCurrentProductStyle] = useState({ photos: [] });
-  const [currentProductSizes, setCurrentProductSizes] = useState({});
+  const [currentProductStyle, setCurrentProductStyle] = useState({ photos: [], skus: {} });
+  const currentProductSizes = currentProductStyle.skus;
```

Comment:
This is preventing the navbar from showing when given a product id that doesn't exist because skus property will be undefined. Will merge for now as this is not a priority error at the time, but keep in mind heading forward if can add a conditional to alleviate this.